tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -32,7 +31,7 @@ parameters:
 
 - name: poolBuild
   type: object
-  default: Small
```

Comment:
Because the time it takes for the repo to build (`npm run ci:build`) in [Small](https://dev.azure.com/fluidframework/internal/_build/results?buildId=86542&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3) vs [Large](https://dev.azure.com/fluidframework/internal/_build/results?buildId=89330&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3) is extremely different, 33m vs 5m. If we're fine letting this pipeline take much longer to run, we can change it back to Small, but I thought Large would be better. If we eventually move to a model where only 1 pipeline does the build and the rest work on its artifacts, this decision will go away 😄 .

Reply:
ah... we are still rebuilding in the pipeline.  That make more sense.

We should switch back to small once (or if) we stop rebuilding.